apps/prairielearn/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "build": "tsc && tscp && compiled-assets build ./assets ./public/build",
-    "dev:no-watch": "ts-node --transpile-only src/server.js",
+    "dev:no-watch": "ts-node src/server.js",
```

Comment:
We specify `ts-node.transpileOnly` in `tsconfig.json`, so the flag is unnecessary.